.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -27,28 +30,39 @@ body:
     validations:
       required: true
   - type: textarea
+    id: expected
     attributes:
       label: Expected behavior
       description: A clear and concise description of what you expected to happen.
   - type: textarea
+    id: prisma-info
     attributes:
       label: Prisma information
       description: Your Prisma schema, Prisma Client queries, ...
       value: |
         <!-- Do not include your database credentials when sharing your Prisma schema! -->
+        ```prisma
+        // Add your schema.prisma
+        ```
+
+        ```ts
+        // Add your code using Prisma Client
+        ```
```

Comment:
Just realizing that this could backfire with a lot of empty code blocks. Let's see how this goes.